<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="extensionsLabel">
             <property name="text">
              <string>Extensions</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="searchPathLabel">
             <property name="text">
              <string>Search path</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="extensionsEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="searchPathEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="notOlderThanCheckBox">
           <property name="text">
            <string>Not older than</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="daysSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="daysLabel">
           <property name="text">
            <string>days</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="sizeFileCheckBox">
           <property name="text">
            <string>Size file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="signsComboBox"/>
         </item>
         <item>
          <widget class="QSpinBox" name="kbytesSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="kbytesLabel">
           <property name="text">
            <string>Kbytes</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="searchResultsLabel">
     <property name="text">
      <string>Search results</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
